content/source/docs/enterprise/api/teams.html.md

```diff
@@ -94,3 +201,7 @@ $ curl \
   --request DELETE \
   https://app.terraform.io/api/v2/teams/257529
 ```
+
+## Available Related Resources
```

Comment:
If this is a new addition please be sure to update the template. /cc @nfagerlund

Reply:
yup yup